ssh_automated_tests/core/user.py

```diff
@@ -16,3 +20,13 @@ def make_connection_data(self):
         return {"hostname": self._address, "port": self._port,
                 "username": self._name, "key": self._key}
 
+
+class SshUserParser(object):
```

Comment:
@extsoft Ok, I have several ideas:
1) Avoid creating extra classes and create user instance using `ConfgReader` class.

```
    cfg_reader = ConfigReader(RawFile('config_draft.properties'))

    user = SshUser(name=cfg_reader.get_value('connection', 'user_name'),
                   port=int(cfg_reader.get_value('connection', 'port')),
                   key=cfg_reader.get_value('connection', 'key'),
                   address=cfg_reader.get_value('connection', 'ip_address'))
```
2) Use `SshUserParser` to create user and hide some work inside it`s method.

```
    parser = SshUserParser(ConfigReader(RawFile('config_draft.properties')))

    user = parser.parse_user('connection', 'user_name', 'port', 'key', 'ip_address')
```

3) Rework previous way to hide all parameters inside class method.

```
parser = SshUserParser(ConfigReader(RawFile('config_draft.properties')))

user = parser.parse_user()
```


4)  Change the principle of parsing inside `ConfigReader` class to use built in `open()` and `*.readline()` functions and for each loops to parse configuration file. This can avoid adding `[connection]` section to configuration file. Code in `conftest.py` will look similar to previous.
I think it will be possible to use other file extensions for configurations (for example json, ini, yaml, xml or txt).

```
    parser = SshUserParser(ConfigReader(RawFile('config_draft.txt')))

    user = parser.parse_user()
```